src/components/subscriptions/RemindUserButton.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LicenseRemindModal from '../../containers/LicenseRemindModal';
+import ActionButtonWithModal from '../ActionButtonWithModal';
+
+const RemindUserButton = ({
+  onSuccess,
+  onClose,
+  pendingUsersCount,
+  isBulkRemind,
+}) => (
+  <ActionButtonWithModal
+    buttonLabel="Remind all"
+    buttonClassName="btn btn-primary float-right"
```

Comment:
Remind all button is not styled correctly, I fixed it in my PR @ https://github.com/edx/frontend-app-admin-portal/pull/268/files#diff-26fd841adaaa4ef93bd203aeee34a7c9L19 
You should not need to do anything here, rebasing your branch with master once my PR merges should be enough. Leaving this comment just to remind you to validate the UI after the rebase.

Reply:
Yeah, that is the reason I didn't make these changes to avoid conflicts.